unyt/mpl_interface.py

```diff
@@ -18,6 +18,7 @@
         ConversionInterface,
         AxisInfo,
         registry,
+        ConversionError,
```

Comment:
It looks like importing this broke support for matplotlib 2.2.4. Can you mock `ConversionError` in the `except ImportError` block below?